src/Link.js

```diff
@@ -17,31 +16,26 @@ const buttonStyles = {
   appearance: 'none'
 }
 
-const hoverColor = system({
```

Comment:
I'm not really sure what this was doing. I couldn't find any color emitted for `:hover` on the links. It seems like for normal links the color is the same on hover, but if this needs to be added back for another reason, let me know!

Reply:
This adds a custom styled system prop to the component so that users of the component can configure what color the links should be on hover. I think we should put this back in - there were a few places where we were changing the hover color intentionally - I believe the primer.style site still uses it!